test/ao/sparsity/test_composability.py

```diff
@@ -377,6 +377,59 @@ def test_q_prep_fx_before_s_prep(self):
         )
         self.assertGreaterAlmostEqual(cur_sparsity, sparse_config[0]["sparsity_level"])
 
+    def test_q_prep_fx_s_prep_ref_conv(self):
```

Comment:
same for these, can we add some docstrings, it might get confusing at a later time